.github/workflows/stdlib_dependabot.yml

```diff
@@ -0,0 +1,70 @@
+name: Ballerina Stdlib Dependabot
+
+on: 
+    workflow_dispatch:
+    schedule:
+        - cron: '30 18 * * *'
+jobs:
+    dependabot:
+        name: Stdlib Dependabot
```

Comment:
Shall we add the owner check here similar to what we did in the other workflows?

Reply:
Will do